acord/models/guild.py

```diff
@@ -159,7 +167,7 @@ class Guild(pydantic.BaseModel, Hashable):
     vanity_url_code: Optional[str]
     """ the vanity url code for the guild """
 
-    verification_level: int
+    verification_level: VerificationLevel
```

Comment:
:class:\`VerificationLevel\`

Reply:
Fixed in https://github.com/Mecha-Karen/acord/pull/11/commits/b8f6ab529a1024a69e5cbc142934dbb22ba31b2c